HEMCO/Extensions/hcox_seasalt_mod.F90

```diff
@@ -290,23 +383,170 @@ SUBROUTINE HCOX_SeaSalt_Run( am_I_Root, ExtState, HcoState, RC )
+          QSNOWICE_MYI = QS * SLNT_MYI * MULTI(I,J) / 1000d0
+         ELSE
+          QSNOWICE_FYI = 0.0d0
+          QSNOWICE_MYI = 0.0d0
+         ENDIF
+       ENDIF
+       ! End of added blowing snow section
+       !-----------------------------------------------------------------
+
        ! Do for accumulation and coarse mode, and Marine POA if enabled
        DO N = 1,Inst%NSALT
 
-          ! Reset values for SALT and SALT_N
+          ! Reset values for SALT, SALT_N, and SNOWSALT
           SALT   = 0d0
           SALT_N = 0d0
-
+          SNOWSALT = 0d0
+
+          ! update seasalt from blowing snow - huang 1/4/18
+          IF (( Inst%EmitSnowSS ) .and. ( N .LT.3 )) THEN
+             IF ( HcoState%Grid%YEDGE%Val(I,J) .lt. 0 ) THEN
+                ! Southern Hemisphere
+                SALT = SALT + HcoState%TS_EMIS * A_M2 &
+                     * ( QSNOWICE_FYI * SUM( Inst%F_DI_S_FYI(:,N) ) +  &
+                         QSNOWICE_MYI * SUM( Inst%F_DI_S_MYI(:,N) ) ) * DDSNOW
+
+                SNOWSALT = SNOWSALT + HcoState%TS_EMIS * A_M2 &
+                     * ( QSNOWICE_FYI * SUM( Inst%F_DI_S_FYI(:,N) ) + & 
+                         QSNOWICE_MYI * SUM( Inst%F_DI_S_MYI(:,N) ) ) * DDSNOW
+
+                SALT_N = SALT_N + HcoState%TS_EMIS * A_M2 &
+                     * ( QSNOWICE_FYI * SUM( Inst%F_DN_S_FYI(:,N) ) + &
+                         QSNOWICE_MYI * SUM( Inst%F_DN_S_MYI(:,N) ) ) * DDSNOW
+              ELSE
+                ! Northern Hemisphere
+                SALT = SALT + HcoState%TS_EMIS * A_M2 &
+                    * ( QSNOWICE_FYI * SUM( Inst%F_DI_N_FYI(:,N) ) + &
+                        QSNOWICE_MYI * SUM( Inst%F_DI_N_MYI(:,N) ) ) * DDSNOW
+
+                SNOWSALT = SNOWSALT + HcoState%TS_EMIS * A_M2 &
+                    * ( QSNOWICE_FYI * SUM( Inst%F_DI_N_FYI(:,N) ) + &
+                        QSNOWICE_MYI * SUM( Inst%F_DI_N_MYI(:,N) ) ) * DDSNOW 
+
+                SALT_N = SALT_N + HcoState%TS_EMIS * A_M2 &
+                     * ( QSNOWICE_FYI * SUM( Inst%F_DN_N_FYI(:,N) ) + &
+                         QSNOWICE_MYI * SUM( Inst%F_DN_N_MYI(:,N) ) ) * DDSNOW
+              ENDIF
+              ! add bromine blowing snow 
+              IF ( Inst%CalcBr2 ) THEN
+                IF ( HcoState%Grid%YEDGE%Val(I,J) .lt. 0 ) THEN
+                SSA_Br2 =  SSA_Br2 + HcoState%TS_EMIS * A_M2 &
+                        * (QSNOWICE_FYI * SUM( Inst%F_DI_S_FYI(:,N) ) + &
+                           QSNOWICE_MYI * SUM( Inst%F_DI_S_MYI(:,N) ) ) * DDSNOW &
+                        * 0.00223d0 * 0.7d0 / 2.0d0
+                ELSE
+                SSA_Br2 =  SSA_Br2 + HcoState%TS_EMIS * A_M2 &
+                        * (QSNOWICE_FYI * SUM( Inst%F_DI_N_FYI(:,N) ) + &
+                           QSNOWICE_MYI * SUM( Inst%F_DI_N_MYI(:,N) ) ) * DDSNOW &
+                        * 0.00223d0 * 0.7d0 / 2.0d0
+                ENDIF
+              ENDIF
+          ENDIF
```

Comment:
The bromine option in the sea salt extension was retired in 12.9. To accomodate that change I excluded this section in the blowing snow update in GEOS-Chem 13. @jaegle, can you confirm that this is okay? Are there other places that need editing as well? Also, are you okay with the bromine option being retired in the first place?

Reply:
Yes, you can remove the CalcBr2 option. 

A few more places that need editing:

1) the variable AGE should be included in the PRIVATE statement in hcox_seasalt_mod.F90 (it wasn't in the patch I submitted)

2) I noticed that when using HEMCO standalone to calculate blowing snow emissions, surface pressure was set at 101325.0 Pa. I use the variable HcoState%Grid%PSFC%Val( I, J ):
         ! Surface pressure at grid box (I,J). Convert from [Pa] to [hPa]
         PRESS = HcoState%Grid%PSFC%Val( I, J ) /100d0
should a difference variable be used?

Reply:
1. I made that change. Thanks for noticing that.
2. Yes, using that variable should be fine. What you saw in 12.7 may be a bug that has since been fixed where PS was not used to set pressure in the absence of PSFC listed as an input.